etc/adminhtml/system.xml

```diff
@@ -958,8 +958,23 @@
                     <label>Enable Indexing Queue</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                     <comment>
-                        <model>Algolia\AlgoliaSearch\Model\Config\EnableQueueComment</model>
+                        <![CDATA[
+                            If enabled, all indexing operations (add, remove & update operations) will be done asynchronously using the CRON mechanism.<br><br/>
+                            <span class="algolia-config-warning">&#9888;</span> Enabling this option is recommended in production or if your store has a lot of products.
+                        ]]>
+                    </comment>
+                </field>
+                <field id="cron_expr" translate="label comment" type="text" sortOrder="20" showInDefault="1" showInWebsite="0" showInStore="0">
```

Comment:
Perhaps we can make this field dependent on a Yes/No flag that enables/disabled "built in cron" per my earlier comment. We could make it so that leaving this blank disables the cron as well but that might break your validation and perhaps be less intuitive?